Documentation/compatibility/wpf-selector-selectionchanged-and-selectedvalue.md

```diff
@@ -0,0 +1,56 @@
+## Selector SelectionChanged event and SelectedValue property
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+When its selection changes, a `T:System.Windows.Controls.Primitives.Selector` now updates the value of its
+`P:System.Windows.Controls.Primitives.Selector.SelectedValue` property before raising the
+`E:System.Windows.Controls.Primitives.Selector.SelectionChanged` event.
+This makes the SelectedValue property consistent with the other selection properties (SelectedItem and SelectedIndex), which are updated before raising the event.
+
+Prior to this fix, the update to SelectedValue happened before the event in most cases, but happened
```

Comment:
Instead of "Prior to this fix", "In the .NET Framework 4.7 and earlier versions"
but happened --> but it happened